src/Agent.ts

```diff
@@ -404,7 +404,7 @@ export class Agent {
     body?: Record<string, unknown>
   ): Promise<T> {
     const { data: response } = await Agent.call<MangadexApiResponse<T>, 'json'>(
-      `api${!url.startsWith('/') && '/'}${url}`,
+      `${!url.startsWith('/') && '/'}${url}`,
```

Comment:
You can just pass url to the first argument, `Agent.call `already contains that code inside.